.github/workflows/upload-last-push/linux/configure.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# You can use your interpreter of choice (bash, sh, zsh, ...)
+
+# For any question please contact with us in:
+#   - https://github.com/KratosMultiphysics/Kratos
+
+# Optional parameters:
+# You can find a list will all the compiation options in INSTALL.md or here:
+#   - https://github.com/KratosMultiphysics/Kratos/wiki/Compilation-options
+
+add_app () {
+    export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}$1;"
+}
+
+# Set variables
+export KRATOS_SOURCE="${KRATOS_SOURCE:-${PWD}}"
+export KRATOS_BUILD="${KRATOS_SOURCE}/build"
+export KRATOS_APP_DIR="${KRATOS_SOURCE}/applications"
+
+# Set build type
+export KRATOS_BUILD_TYPE=Release
+export PYTHON_EXECUTABLE="/usr/bin/python3.6"
+export KRATOS_INSTALL_PYTHON_USING_LINKS=ON
+
+# Set applications to compile
+add_app ${KRATOS_APP_DIR}/ExternalSolversApplication;
+add_app ${KRATOS_APP_DIR}/EigenSolversApplication;
+add_app ${KRATOS_APP_DIR}/StructuralMechanicsApplication;
+add_app ${KRATOS_APP_DIR}/FluidDynamicsApplication;
+add_app ${KRATOS_APP_DIR}/MeshMovingApplication;
+add_app ${KRATOS_APP_DIR}/DEMApplication;
+add_app ${KRATOS_APP_DIR}/CSharpWrapperApplication;
+add_app ${KRATOS_APP_DIR}/MetisApplication;
+add_app ${KRATOS_APP_DIR}/TrilinosApplication;
+add_app ${KRATOS_APP_DIR}/ShapeOptimizationApplication;
+add_app ${KRATOS_APP_DIR}/CoSimulationApplication;
+add_app ${KRATOS_APP_DIR}/CableNetApplication;
+add_app ${KRATOS_APP_DIR}/RANSApplication;
+add_app ${KRATOS_APP_DIR}/MappingApplication;
+add_app ${KRATOS_APP_DIR}/FSIApplication;
+add_app ${KRATOS_APP_DIR}/MeshingApplication;
```

Comment:
Can we deliver with the MMG as we are going to do in the CI?

Reply:
I guess if we use the same configure as for the CI then it would be straight forward

Reply:
Sure @loumalouomega , I need at least the apps in the GiD GUI, but we can include also the others

Reply:
Vicente I thought we cannot distribute MMG because of their license?

Reply:
We can i we reach an agreement with them, and I am sure we can

Reply:
is it really that easy?
They are lgpl afaik, they would basically have to change their license or not?

Reply:
If you reach an agreement with the owners of the licence you ca do that. GMSH is GPL3 and they reach agreements to be included as private software http://gmsh.info/#Licensing

Reply:
ok cool :+1: